### PR TITLE
Updating data.json content from airtable

### PR DESCRIPTION
**Pull request by SupeDeDupe**
1st attempt at updating the content using this new method.

Scott and Allison agreed changes on their end are ready to be merged, and new data from the translation sheet is ready to be used in the feedback_submitted page.

### CODE REVIEW

File: data/data.json [hunk 1]
```diff
@@ -4081,10 +4089,28 @@
...
       "Safe to Delete?": []
+    },
+    {
+      "section": "feedback",
+      "key": "submitted",
+      "English": "Thank you for taking the time to give us feedback. We are listening and constantly improving.",
+      "French": "Merci d’avoir pris le temps de nous faire part de vos commentaires. Nous sommes à l’écoute et nous nous améliorons constamment.",
+      "id": "feedback.submitted",
+      "Used by code": [],
+      "Safe to Delete?": []
```
Comment (SupeDeDupe) [inline data/data.json:4083]: We can probably delete the "Used by code" and "Safe to Delete" columns in translations in Airtable.

Comment (sastels) [reply data/data.json:4083]: But is it safe to delete the "Safe to Delete" column? 🤔

File: data/data.json [hunk 2]
```diff
@@ -5383,5 +5339,5 @@
     }
   ],
   "errors": [],
-  "timestamp": 1549485129335
+  "timestamp": 1549907642205
```
Comment (sastels) [inline data/data.json:5342]: huh, at some point we should probably save the timestamp as a human readable string

**Review by sastels — APPROVED** [review]
LGTM!

**Pull request merged**